src/data/faq.json

```diff
@@ -159,7 +159,7 @@
                                 "active": false,
                                 "textblock": [
                                     "At the moment, many families are tested at the airport after returning from their holidays. For each family member that is being tested, they get a leaflet that displays a PCR-Test QR code.",

```

Comment:
"At the moment" was part of a historical snapshot last year. I suggest instead to remove the example of "airport" (I not sure whether this is really the mainstream use-case these days) and just talk in general about families whose do tests for their individual family members of whom some do not have their own smartphones.

The German text is a bit different, but the general comment applies there as well.

Reply:
I don't really see an issue with bringing an example for when this could take place, but the English translation doesn't really mean the same as the German original. I'd suggest to word the English version to make it more clear that it is an example and remove any reference of time. 

Maybe change it to: "Families who are tested at the airport after a vacation, for example, are often given a leaflet for each test that includes a PCR test QR code."

What do you think about this?

Reply:
@larswmh 

The original sentence didn't seem to make much sense because it sounded like the test took place at the airport (in Germany) when they landed after returning from vacation. They would need to get tested before they got on a plane I believe, not after they had already travelled on a plane and disembarked.

If you change your suggestion to "Families who are tested at an airport before returning from vacation, for example, are often given a leaflet for each test that includes a PCR test QR code." that would make sense to me.

I was trying to make the use-case more generic, but if the airport example can be made logical and understandable, then I'm fine with that.

Reply:
Thanks for the suggestion, it has been applied in the latest commit with a minor change to the German text so it has the same meaning